go.mod

```diff
@@ -26,22 +25,20 @@ require (
 	github.com/decred/dcrd/wire v1.2.0
 	github.com/decred/dcrwallet/wallet v1.1.0
 	github.com/decred/slog v1.0.0
-	github.com/dgraph-io/badger v1.5.5-0.20190124015931-be03c12d2da2
+	github.com/dgraph-io/badger v1.5.5-0.20190214192501-3196cc1d7a5f
 	github.com/dgryski/go-farm v0.0.0-20180109070241-2de33835d102 // indirect
 	github.com/didip/tollbooth v4.0.0+incompatible
 	github.com/didip/tollbooth_chi v0.0.0-20170928041846-6ab5f3083f3d
 	github.com/dustin/go-humanize v1.0.0
-	github.com/go-chi/chi v3.3.3+incompatible
-	github.com/go-chi/docgen v1.0.2
-	github.com/golang/snappy v0.0.0-20180518054509-2e65f85255db // indirect
+	github.com/go-chi/chi v4.0.1+incompatible
+	github.com/go-chi/docgen v1.0.5
```

Comment:
docgen 1.0.5 now has a go.mod, and is updated to use chi 4.0.0 imports and newer golang.org/x packages